libs/components/phone-field/src/lib/modules/shared/forms-utility.ts

```diff
@@ -0,0 +1,9 @@
+// Need to add the following to classes which contain static methods.
+// See: https://github.com/ng-packagr/ng-packagr/issues/641
+// @dynamic
+export class SkyFormsUtility {
+  /** Coerces a data-bound value (typically a string) to a boolean. */
+  public static coerceBooleanProperty(value: any): boolean {
```

Comment:
Why not use `@angular/cdk`'s version instead of creating a new class? You'll have to add the peer dependency.
https://material.angular.io/cdk/coercion/api